src/layouts/AppLayout.vue

```diff
@@ -2,20 +2,39 @@
 import { computed } from 'vue';
 import { RouterView, useRoute } from 'vue-router';
 
+import useAuthStore from '@/store/auth';
 import AppNavbar from '@/components/AppNavbar.vue';
 
 const route = useRoute();
 
+const authStore = useAuthStore();
+
 const pageTitle = computed(() => route.meta.title);
+
+async function logout(): Promise<void> {
+  await authStore.signOut();
+}
 </script>
 
 <template>
   <div class="layout-container">
     <app-navbar />
     <div class="layout-container-content">
-      <h1 class="title">
-        {{ pageTitle }}
-      </h1>
+      <div class="header">
+        <h1 class="title">
+          {{ pageTitle }}
+        </h1>
+        <div class="profile-container">
+          <i class="fa-solid fa-circle-user profile-icon"></i>
```

Comment:
This should close like this '/>' unless you're wrapping something, are you wrapping something?